extensions/skyportal/static/js/components/ZTFAlertsByCoords.jsx

```diff
@@ -0,0 +1,372 @@
+          colSpan={colSpan}
+        >
+          <Grid
+            container
+            direction="row"
+            spacing={3}
+            justify="center"
+            alignItems="center"
+          >
+            <Grid item>
+            <ThumbnailList
+              thumbnails={thumbnails}
+              ra={alertData.candidate.ra}
+              dec={alertData.candidate.dec}
+              displayTypes={["new", "ref", "sub"]}
+            />
+            </Grid>
+          </Grid>
+        </TableCell>
+      </TableRow>
+    );
+  };
+
+  const options = {
+    selectableRows: "none",
+    expandableRows: true,
+    expandableRowsOnClick: true,
+    renderExpandableRow: renderPullOutRow,
+    elevation: 1,
+    sortOrder: {
+      name: "jd",
+      direction: "desc",
+    },
+  };
+
+  const columns = [
+    {
+      name: "objectId",
+      label: "Object ID",
+      options: {
+        filter: true,
+        sort: true,
+        sortDescFirst: true,
+      },
+    },
+    {
+      name: "candid",
+      label: "candid",
+      options: {
+        filter: false,
+        sort: true,
+      },
+    },
+    {
+      name: "jd",
+      label: "JD",
+      options: {
+        filter: false,
+        sort: true,
+        sortDescFirst: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(5),
+      },
+    },
+    {
+      name: "ra",
+      label: "RA",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => ra_to_hours(value, ":"),
+      },
+    },
+    {
+      name: "dec",
+      label: "Dec.",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => dec_to_dms(value, ":"),
+      },
+    },
+    {
+      name: "drb",
+      label: "drb",
+      options: {
+        filter: true,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "fid",
+      label: "fid",
+      options: {
+        filter: true,
+        sort: true,
+      },
+    },
+    {
+      name: "mag",
+      label: "mag",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(3),
+      },
+    },
+    {
+      name: "emag",
+      label: "e_mag",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(3),
+      },
+    },
+    {
+      name: "programid",
+      label: "programid",
+      options: {
+        filter: true,
+        sort: true,
+      },
+    },
+  ];
+
+  if (alerts === null || !isFetched) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+  if (isString(alerts)) {
+    return <div>Failed to fetch alert data, please try again later.</div>;
```

Comment:
Who said it'll work next time lol? :)

```suggestion
    return <div>Failed to fetch alert data.</div>;
```